archive.istio.io/public/404.html

```diff
@@ -13,42 +13,50 @@
 
     <title>Page Not Found</title>
 
-    <link rel="shortcut icon" href="/v0.5/favicons/favicon.ico">
-    <link rel="apple-touch-icon" href="/v0.5/favicons/apple-touch-icon-180x180.png" sizes="180x180">
-    <link rel="icon" type="image/png" href="/v0.5/favicons/favicon-16x16.png" sizes="16x16">
-    <link rel="icon" type="image/png" href="/v0.5/favicons/favicon-32x32.png" sizes="32x32">
-    <link rel="icon" type="image/png" href="/v0.5/favicons/android-36x36.png" sizes="36x36">
-    <link rel="icon" type="image/png" href="/v0.5/favicons/android-48x48.png" sizes="48x48">
-    <link rel="icon" type="image/png" href="/v0.5/favicons/android-72x72.png" sizes="72x72">
-    <link rel="icon" type="image/png" href="/v0.5/favicons/android-96x196.png" sizes="96x196">
-    <link rel="icon" type="image/png" href="/v0.5/favicons/android-144x144.png" sizes="144x144">
-    <link rel="icon" type="image/png" href="/v0.5/favicons/android-192x192.png" sizes="192x192">
+    <link rel="shortcut icon" href="./v0.5/favicons/favicon.ico">
```

Comment:
this can’t have ./ - it won’t work as a 404 that way - see my previous fix to it